### PR TITLE
Add safemode ruby gem advisory

### PR DESCRIPTION
**Pull request by skorth**
Please double check, thx.

### CODE REVIEW

File: gems/safemode/CVE-2016-3693.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+gem: safemode
+cve: 2016-3693
+title: Information leakage through templates
+date: 2016-04-20
+url: http://seclists.org/oss-sec/2016/q2/119
+description: |
+  A provisioning template containing `inspect` will expose sensitive
```
Comment (reedloden) [inline gems/safemode/CVE-2016-3693.yml]: This is all Foreman-specific, but the gem has nothing to do with foreman. Can you rewrite this to be specific to what was actually vulnerable for `safemode`?

Comment (skorth) [inline gems/safemode/CVE-2016-3693.yml]: @reedloden suggestion:

"Safemode is initialised with an optional 'delegate' object.
If the delegated object is a Rails controller, 'inspect' could
be called which then exposes all informations about the App,
including routes, secret tokens, caches and so on."

Comment (reedloden) [inline gems/safemode/CVE-2016-3693.yml]: Seems fine. Update the title as well?

**Comment by skorth** [comment]
@reedloden Sorry for the delay -> updated the title.

**Pull request merged**